### PR TITLE
Curate Windows Subsystem for Linux topic

### PR DESCRIPTION
**Pull request by alannt777**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ x Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`. No images yet. Will add later
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/wsl/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# Windows Subsystem for Linux
+
+Windows Subsystem for Linux allows you run native Linux applications on Windows
+
+
+
+- WSL 1 uses a translation layer to translate Linux syscalls to Windows syscalls
+
+- WSL 2 uses lightweight utility virtual machine to run Linux apps
```
Comment (kenyonj) [inline topics/wsl/index.md:17]: Are these new lines needed?
```suggestion
# Windows Subsystem for Linux

Windows Subsystem for Linux allows you run native Linux applications on Windows

- WSL 1 uses a translation layer to translate Linux syscalls to Windows syscalls
- WSL 2 uses lightweight utility virtual machine to run Linux apps
```

**Review by kenyonj — CHANGES_REQUESTED** [review]
I think we can get rid of some of these new lines. LMK what you think.

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Comment by kenyonj** [comment]
Just noticed that the topic title is duplicated
image

I'm going to remove the first line, feel free to open another PR if you have other ideas.

**Comment by alannt777** [comment]
@kenyonj Thank you! I will add some more content in another PR

**Pull request merged**